Makefile

```diff
@@ -0,0 +1,72 @@
+cache:
+	docker exec -it cmp-app bin/console
+
+cache-clear:
+	rm -rf var/cache/*
+	rm -rf var/log/*
```

Comment:
This will also remove the versioned file  `var/log/.gitkeep`. The file is needed because the folder must always exist.

Reply:
Nope, try it or check https://unix.stackexchange.com/a/77313/398062